client/landing/gutenboarding/onboarding-block/acquire-intent/vertical-select/style.scss

```diff
@@ -51,6 +51,7 @@
 	position: relative;
 	display: block;
 	flex: 1;
+	z-index: 2;
```

Comment:
All across Calypso (and Gutenberg) we try to pull these from mixin for easier management (it becomes very unwieldy otherwise fast).

Feels like Gutenboarding should have its own index tho. 🤔 Especially since we can't re-use both mixins from Calypso and Gutenboarding (they have the same name).

Should we just use `.components-popover` z-index from Gutenberg and get back to these later?

https://github.com/Automattic/wp-calypso/blob/master/client/assets/stylesheets/shared/functions/_z-index.scss
https://github.com/WordPress/gutenberg/blob/master/packages/base-styles/_z-index.scss